components/TextFields/tests/snapshot/MDCBaseTextFieldSnapshotTests.m

```diff
@@ -35,6 +37,29 @@ - (void)tearDown {
   [super tearDown];
 }
 
+- (MDCBaseTextField *)createBaseTextFieldInKeyWindow {
+  MDCBaseTextField *textField = [[MDCBaseTextField alloc] initWithFrame:CGRectMake(0, 0, 200, 60)];
+  textField.borderStyle = UITextBorderStyleRoundedRect;
+  UIWindow *keyWindow = [[UIApplication sharedApplication] keyWindow];
+  [keyWindow addSubview:textField];
```

Comment:
I expect that we would want to tear this down so that it does not affect other tests.
In the tear down remove it from the window.

Reply:
In 35b1fb59ea6828b2fbb9ebb17a9b95894a7da2d0 I implement `setUp` and `tearDown` so textfields aren't left just sitting in the window 👍